frontend/src/components/Dashboard/AnnotationDashboard.vue.js

```
@@ -1,201 +1,699 @@
+            });
+            __VLS_elementAsFunction(__VLS_intrinsicElements.i, __VLS_intrinsicElements.i)({
+                ...{ class: ((meta.requires_validation ? 'fas fa-exclamation-triangle' : 'fas fa-check-circle')) },
+            });
+            (meta.requires_validation ? 'Ja' : 'Nein');
+            __VLS_elementAsFunction(__VLS_intrinsicElements.td, __VLS_intrinsicElements.td)({});
+            __VLS_elementAsFunction(__VLS_intrinsicElements.div, __VLS_intrinsicElements.div)({
+                ...{ class: ("btn-group btn-group-sm") },
+            });
+            __VLS_elementAsFunction(__VLS_intrinsicElements.button, __VLS_intrinsicElements.button)({
+                ...{ onClick: (...[$event]) => {
+                        if (!(!((__VLS_ctx.loadingSensitiveMeta))))
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)